ecs-agent/tmds/utils/netconfig/netconfig_windows.go

```diff
@@ -27,6 +30,6 @@ func NewNetworkConfigClient() *NetworkConfigClient {
 
 // DefaultNetInterfaceName returns the device name of the first default network interface
 // available on the instance. This is only supported on linux as of now.
-func DefaultNetInterfaceName() (string, error) {
-	return "", errors.New("Not supported on windows")
+func DefaultNetInterfaceName(unknown interface{}) (string, error) {
+	return "", errors.New("not supported on windows")
```

Comment:
Thanks for fixing this!

nit (non-blocking): should we also make the error message un-capitalized for the unsupported file as well? happy for this to be done in a follow up PR/can also include this in my PR

Reply:
Will do this in a follow up PR.